### ISSUE TITLE
borgbackup does not work

### ISSUE THREAD

**Issue by real-or-random** [termux/termux-packages#4711]
**Problem description**
```
 raise DistributionNotFound(req, requirers) pkg_resources.DistributionNotFound: The 'borgbackup==1.1.10' distribution was not found and is required by the application
```

**Steps to reproduce**
 - Install borgbackup on a fresh termux.
-  Run `borg`

**Expected behavior**
Borg works

**Additional information**
Here's a full trace including termux-info
https://gist.github.com/real-or-random/2746b430487d4e9ba482cfa0b5c3712b

**Comment by xeffyr** [comment]
Should be fixed in `1.1.10-5`.

**Comment by real-or-random** [comment]
Thanks! This has fixed the problem.
